### PR TITLE
Fix Typing Issues

### PR DESCRIPTION
**Pull request by jkosh44**
Hi Slippi team,
I wanted to try and start contributing to this project. I thought fixing some typing FIXMEs would be a good place to start to familiarize myself with the Repo. I didn't see any CONTRIBUTING.MD so please let me know if I need to make any changes.

### CODE REVIEW

**Comment by JLaferri** [comment]
I think this PR might have been for an older version of the codebase. Should I close this or is it still somewhat valid?

**Comment by jkosh44** [comment]
Yeah looks like the codes been updated since opening. Feel free to close this.

**Comment by jkosh44** [comment]
Actually going through the diff there are still some changes that are applicable. I've pushed the merge conflicts, but it won't let me re-open the PR because I did a force push. Do I open a new one or can someone else re-open?

File: src/stats/overall.ts [hunk 1]
```diff
@@ -187,8 +186,7 @@ function generateInputs(game: SlippiGame): Array<PlayerInput> {
 
     // Do the same for c-stick
     const prevCstickRegion = getJoystickRegion(prevPlayerFrame.cStickX, prevPlayerFrame.cStickY);
-    // FIXME: stop using any. it is not clear what type this is. cstickX does not exist on the type PreFrameUpdateType
-    const currentCstickRegion = getJoystickRegion((playerFrame as any).cStickX, (playerFrame as any).cStickY);
```
Comment (jkosh44) [inline src/stats/overall.ts]: Pretty sure these comments are wrong because I do see those fields in PreFrameUpdateType

**Comment by JLaferri** [comment]
> Actually going through the diff there are still some changes that are applicable. I've pushed the merge conflicts, but it won't let me re-open the PR because I did a force push. Do I open a new one or can someone else re-open?

I'll close this, feel free to re-open a new PR

**Pull request closed without merging**